sms_sponsorship/models/recurring_contract.py

```diff
@@ -179,6 +179,12 @@ def contract_waiting(self):
         self._post_payment_first_month()
         return True
 
+    @api.multi
+    def contract_waiting(self):
```

Comment:
The method is already defined just above.

Reply:
Yeah, right...